pkg/policy/repository.go

```diff
@@ -536,9 +536,15 @@ func (p *Repository) GetRulesMatching(lbls labels.LabelArray) (ingressMatch bool
 			if len(r.Ingress) > 0 {
 				ingressMatch = true
 			}
+			if len(r.IngressDeny) > 0 {
+				ingressMatch = true
+			}
 			if len(r.Egress) > 0 {
 				egressMatch = true
 			}
+			if len(r.EgressDeny) > 0 {
+				egressMatch = true
+			}
```

Comment:
Could we extend the unit tests in `pkg/policy/repository_deny_test.go` to cover this?